iml-gui/crate/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+version = "0.1.0"
+name = "iml-gui"
+repository = "https://github.com/whamcloud/integrated-manager-for-lustre"
+authors = ["IML Team <iml@whamcloud.com>"]
+description = "The IML GUI"
+categories = ["wasm", "rust", "tailwindcss", "seed"]
+license = "MIT"
+readme = "./README.md"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
```

Comment:
Likewise, "cdylib" means "A dynamic system library". Does it make sense for wasm?

Reply:
Yes, it's overloaded for Rust / wasm. Ref: https://rustwasm.github.io/wasm-pack/book/tutorials/npm-browser-packages/template-deep-dive/cargo-toml.html#1-crate-type